src/editor/ToolBarItem.tsx

```diff
@@ -23,8 +23,6 @@ function ToolBarItem({ label, children }: ToolBarItemProps) {
     setAnchorEl(null);
   };
 
-  // modify incoming children, so that each menu item
-  // will also call `closeMenu` on click
```

Comment:
komentarz mogles zostawic xd ja to dopisalem bo troche nieoczywisty ten kod na pierwszy rzut oka